docs/pages/inboxes/device-sync.md

```diff
@@ -0,0 +1,79 @@
+# Enable device sync for apps built with XMTP
```

Comment:
Is "device sync" the name we want to use for this feature? Just making sure it works for folx!